templates/devspace.yaml.tpl

```diff
@@ -94,6 +94,13 @@ vars:
 deployments:
   app:
     namespace: ${DEVENV_DEPLOY_NAMESPACE}
+    resources:
+      requests:
+        cpu: 1
+        memory: 1Gi
```

Comment:
Do we know how this behaves in a regular devenv environment when multiple services use this as their default? The change looks good, but my only worry is that by setting this as a minimum we might limit how many small services people could run at one time.

Reply:
I think the question becomes, how many devspace containers do we expect folks to have running at once?

Reply:
We talked about it offline - to be clear this only affects deploying apps into the devenv from source (e.g., `devenv apps run`).